osu.Game/Overlays/Mods/ModButton.cs

```diff
@@ -63,19 +67,8 @@ private int selectedIndex
         private Color4 buttonColour;
         public Color4 ButtonColour
         {
-            get
-            {
-                return buttonColour;
-            }
-            set
-            {
-                if (value == buttonColour) return;
-                buttonColour = value;
-                foreach (ModIcon icon in iconsContainer.Children)
-                {
-                    icon.Colour = value;
-                }
-            }
+            get { return buttonColour; }
+            set { buttonColour = value; }

```

Comment:
it's needable, because we need that colour after switching back from selected state